WordPress/src/main/res/drawable/ic_site_error.xml

```diff
@@ -0,0 +1,96 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="224dp"
```

Comment:
We can optimize this vector drawable by using compound paths in the imported SVG.  The resulting XML would be this:
``` xml
<vector
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:height="116dp"
    android:width="226dp"
    android:viewportHeight="116"
    android:viewportWidth="226" >

    <path
        android:fillColor="@color/grey"
        android:pathData="M108.8,71.7c0-1.5,0.5-2.5,1-3s1.5-1,3-1c1.5,0,2.5,0.5,3,1s1,1.5,1,3c0,1-0.5,2-1,3s-1.5,1-3,1 c-1.5,0-2.5-0.5-3-1S108.8,73.2,108.8,71.7z M115.8,63.6h-5.5l-1-22.7h8.1L115.8,63.6z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_20"
        android:pathData="M180.7,99.4l-21.8-0.1h-16.5l-16.5,0.1c-11,0.1-22,0.2-33,0.8c-0.1,0.1-0.1,0.1-0.2,0.4c0,0.1,0.1,0.2,0.2,0.2 c11,0.5,22,0.7,33,0.8l16.5,0.1h16.5l21.9-0.1L180.7,99.4z M224.9,100.1c-5.2-0.2-10.4-0.3-15.7-0.4l-0.1,1.6 c5.3-0.1,10.5-0.2,15.8-0.4c0.2,0,0.4-0.2,0.4-0.4S225.1,100.1,224.9,100.1z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M182.6,68.1h0.8l-0.1-1.2l-2.8-25.3l2.2-4.9l3.6,4.2l1.4,12.4l1.5,12.6l0.3,2.2h0.7l-0.2-2l-1.3-12.8l-1.4-12.8 c0-0.1,0-0.2-0.1-0.3l-4.1-4.9c-0.1-0.1-0.1-0.2-0.2-0.2c-0.3-0.1-0.6,0-0.7,0.3l-2.6,5.7v0.3l2.9,25.5L182.6,68.1z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M186.6,40.9c-0.2,0.6-0.6,1.2-1,1.7c-0.1,0.2-0.4,0.2-0.6,0.1h-0.1c-0.5-0.3-1-0.6-1.5-1l0.8-0.1l-1.5,1.5 l-0.1,0.1c-0.2,0.1-0.4,0.1-0.6,0l-1.6-1.5c-0.1-0.1-0.1-0.2,0-0.3c0.1-0.1,0.2-0.1,0.2,0l1.9,1l-0.7,0.1c0.4-0.6,0.9-1.1,1.4-1.6 c0.2-0.2,0.5-0.2,0.8-0.1c0.5,0.3,1,0.7,1.4,1.1l-0.7,0.1c0.5-0.5,1-0.9,1.6-1.2C186.5,40.7,186.6,40.7,186.6,40.9z M181.4,38.4 c1.1,0.2,2.2-0.1,3.1-0.6l-1.8-2.2L181.4,38.4z M195,68.1V49.8c0-0.7,0.3-1.4,0.9-1.9c0.6-0.5,1.3-0.8,2.1-0.7 c1.5,0,2.7,1.3,2.7,2.8v18.1h1.1V50c0.1-1-0.3-2.1-1-2.8c-0.7-0.8-1.7-1.2-2.8-1.2c-0.5,0-1,0-1.5,0.2s-0.9,0.5-1.3,0.8 c-0.8,0.7-1.3,1.7-1.3,2.7v18.4H195z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M197.1,46.2c1.2,0.1,2.4-0.7,2.8-1.8c0.5-1.1,0.6-2.3,0.5-3.5c-0.1-1.2-0.5-2.3-1-3.4s-1.2-2.1-2-3l0.9-0.1 c-0.8,1.7-1.7,3.3-2.3,5c-0.3,0.8-0.6,1.7-0.7,2.6c-0.1,0.9,0,1.8,0.5,2.5v0.2h-0.2c-0.7-0.7-1.1-1.7-1-2.7c0.1-1,0.2-2,0.5-2.9 c0.6-1.8,1.4-3.5,2.2-5.2c0.1-0.3,0.4-0.4,0.7-0.3c0.1,0,0.1,0.1,0.2,0.1c1.8,1.9,3,4.4,3.3,7.1c0.1,1.4-0.1,2.7-0.8,3.9 c-0.4,0.6-0.9,1.1-1.5,1.4C198.6,46.4,197.9,46.4,197.1,46.2C197.1,46.3,197.1,46.3,197.1,46.2C197,46.2,197,46.2,197.1,46.2z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M200.9,40c-0.4,0.4-0.9,0.6-1.4,0.6c-0.6,0.1-1.1,0-1.6-0.3s-0.8-0.5-1.2-0.7c-0.4-0.2-0.8-0.2-1.2-0.4 c-0.1,0-0.2-0.1-0.1-0.2v-0.1c0-0.1,0-0.1,0.1-0.1c0.5-0.2,1.1-0.3,1.6-0.2c0.5,0.1,1,0.3,1.4,0.6c0.3,0.2,0.7,0.3,1.1,0.3 c0.5,0,1,0,1.4,0.2C200.9,39.9,201,39.9,200.9,40z M211.4,68.1C211,77,210.5,86,210,94.9l-0.4,6.7l-0.2,3.4 c-0.1,1.2-0.6,2.3-1.3,3.3c-1.5,1.9-3.9,2.9-6.3,2.7h-13.4c-2.4,0.2-4.8-0.8-6.4-2.7c-0.8-1-1.3-2.1-1.4-3.4 c-0.1-1.2-0.1-2.3-0.1-3.4l-0.2-6.7L179.4,68c0-0.2,0.2-0.4,0.4-0.5l12.8,0.1c4.3,0,8.5,0.1,12.8,0.1c0.1,0,0.2,0.1,0.2,0.2 s-0.1,0.2-0.2,0.2c-4.3,0.1-8.5,0.1-12.8,0.1l-12.8,0.1l0.4-0.5l1.1,26.8l0.3,6.7c0.1,1.1,0.1,2.3,0.2,3.3c0.1,1,0.5,2,1.2,2.8 c0.6,0.8,1.5,1.4,2.4,1.8c1,0.4,2,0.5,3.1,0.5h13.4c2.1,0.1,4.1-0.7,5.5-2.3c0.6-0.8,1-1.8,1.1-2.8l0.2-3.3l0.5-6.7 c0.6-8.9,1.3-17.8,2.1-26.8c0-0.1,0-0.1,0.1-0.1V68.1z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M185.8,75.2c0.4,1.5,0.6,3.1,0.7,4.7s0.3,3.1,0.4,4.7c0.2,3.1,0.4,6.2,0.3,9.4c0,0.1-0.1,0.2-0.2,0.2 s-0.2-0.1-0.2-0.2c-0.6-3.1-0.7-6.2-1-9.3c-0.1-1.6-0.2-3.1-0.3-4.7s-0.1-3.1,0.1-4.7C185.6,75.1,185.7,75.1,185.8,75.2z" >
    </path>

    <path
        android:fillColor="@color/grey"
        android:pathData="M55.2,58.3l-0.4-32.7L54.7,9.2c0-1.5,0.4-3,1.2-4.3c0.8-1.3,2-2.3,3.4-2.9c0.7-0.3,1.4-0.5,2.2-0.5 c0.7-0.1,1.4-0.1,2.1,0h4.1l32.7-0.1l32.7-0.1h28.7c1.6-0.1,3.2,0.3,4.5,1.2c1.4,0.8,2.4,2.1,3.1,3.5c0.4,0.7,0.6,1.5,0.6,2.3 c0.1,0.4,0.1,0.8,0.1,1.2v25.6l-0.1,65.5c0,0.6-0.5,1.1-1.1,1.1l-39.8-0.2l-19.9-0.2L89.3,101c-0.2,0-0.4-0.2-0.4-0.4 c0-0.2,0.2-0.4,0.4-0.4c6.6-0.1,13.3-0.3,19.9-0.3l19.9-0.2l39.8-0.2l-1.1,1.1l-0.1-65.5V9.5c0-0.3,0-0.6-0.1-0.9 c-0.1-0.6-0.2-1.2-0.5-1.7c-0.5-1-1.3-1.9-2.2-2.6c-0.5-0.3-1-0.5-1.6-0.7c-0.3-0.1-0.5-0.1-0.8-0.1h-29.6l-32.7-0.1L67.5,3.3h-6.1 c-0.6,0.1-1.1,0.2-1.7,0.4c-2.3,0.9-3.8,3.2-3.8,5.7l-0.1,16.4l-0.2,32.5c0,0.1-0.1,0.2-0.2,0.2C55.2,58.5,55.2,58.4,55.2,58.3z" >
    </path>

    <path
        android:fillColor="@color/alert_red"
        android:pathData="M54.3,20.4V9c0-4.5,3.1-8.2,6.9-8.2h102c3.8,0,6.9,3.7,6.9,8.2v11.4H54.3z" >
    </path>

    <path
        android:fillColor="@color/grey_darken_20"
        android:pathData="M173.3,20.4c-10.1,0.8-20.3,0.7-30.4,0.8l-30.4,0.1l-30.4-0.1c-10.1,0-20.3,0-30.4-0.8v-0.5 c10.1-0.8,20.3-0.7,30.4-0.8l30.4-0.1l30.4,0.1c10.1,0,20.3,0,30.4,0.8V20.4z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M39.5,70.5c0.6-1,1.3-2,2-3c0.3-0.5,0.7-1,1.1-1.5c0.4-0.5,0.9-0.9,1.5-1.3c0.6-0.3,1.2-0.5,1.9-0.6 s1.2,0,1.9-0.1H57c0.7,0.1,1.3,0.3,1.9,0.6c0.6,0.3,1.1,0.7,1.5,1.2c0.4,0.5,0.7,1,1.1,1.5c0.7,1,1.3,2,2,3c0.1,0.1,0,0.2-0.1,0.3 c-0.1,0.1-0.2,0-0.3,0c-0.8-0.9-1.5-1.8-2.3-2.8c-0.4-0.4-0.8-0.9-1.1-1.4c-0.4-0.4-0.8-0.7-1.3-1c-0.5-0.2-1-0.4-1.5-0.4H46.3 c-0.5,0.1-1,0.2-1.5,0.5c-0.5,0.2-0.9,0.6-1.3,1s-0.8,0.9-1.1,1.4L40,70.6h-0.1C39.4,70.6,39.4,70.5,39.5,70.5z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M48.8,64.6c-0.1-0.2-0.2-0.5-0.3-0.7c0-0.1,0-0.1-0.1-0.2l-0.1-0.1c-0.1-0.1-0.1-0.1-0.2-0.1h-0.7 c-0.2,0-0.3,0.1-0.4,0.3s0,0.5,0.5,0.6c0.1,0,0.2,0.1,0.1,0.2v0.1c0,0.1,0,0.1-0.1,0.1c-0.3,0.1-0.6,0.1-0.9-0.1 c-0.3-0.2-0.5-0.5-0.6-0.9c0-0.4,0.1-0.7,0.3-1c0.3-0.3,0.6-0.4,1-0.4H48c0.4,0,0.8,0.2,1,0.5c0.1,0.2,0.2,0.3,0.2,0.5v0.5 C49.1,64.1,49,64.4,48.8,64.6C48.8,64.8,48.9,64.7,48.8,64.6z M56.1,64.6c-0.1-0.2-0.2-0.5-0.3-0.7c0-0.1,0-0.1-0.1-0.2l-0.1-0.1 c-0.1-0.1-0.1-0.1-0.2-0.1h-0.7c-0.2,0-0.3,0.1-0.4,0.3c-0.1,0.2,0,0.5,0.5,0.6c0.1,0,0.2,0.1,0.1,0.2v0.1c0,0.1,0,0.1-0.1,0.1 c-0.3,0.1-0.6,0.1-0.9-0.1c-0.3-0.2-0.5-0.5-0.6-0.9c0-0.4,0.1-0.7,0.3-1c0.3-0.3,0.6-0.4,1-0.4h0.7c0.4,0,0.8,0.2,1,0.5 c0.1,0.2,0.2,0.3,0.2,0.5v0.5C56.4,64.1,56.3,64.4,56.1,64.6C56.3,64.7,56.2,64.7,56.1,64.6z M70.5,106l-11.3,0.2l-11.3,0.1 l-22.5,0.2h-2.8c-1,0-2-0.2-2.9-0.6c-1.8-0.9-3.1-2.7-3.5-4.7c-0.1-1-0.1-1.9-0.1-2.9V77.2c0-0.5,0-1,0.1-1.5c0.1-1,0.5-2,1.1-2.8 c1.2-1.7,3.1-2.7,5.2-2.7h47.9c1,0,2,0.2,2.9,0.6c1.8,0.9,3.1,2.7,3.5,4.7c0.1,1,0.1,1.9,0.1,2.9v8.4L76.6,98c0,0.9,0,1.9-0.1,2.8 c-0.1,1-0.5,1.9-1.1,2.7C74.3,105.1,72.4,106.1,70.5,106z M70.5,105.7c1.8,0,3.6-0.9,4.6-2.4c0.5-0.7,0.8-1.6,0.9-2.5 c0.1-0.9,0.1-1.9,0-2.8l-0.1-11.3v-8.4c0-0.9,0-1.8-0.1-2.7c-0.3-1.7-1.4-3.2-3-4c-0.8-0.4-1.6-0.5-2.5-0.5H22.4 c-1.7,0-3.4,0.9-4.4,2.3c-0.5,0.7-0.8,1.5-0.9,2.4C17,76.2,17,76.7,17,77.1v21.1c0,0.9,0,1.8,0.1,2.7c0.3,1.7,1.4,3.2,3,4 c0.8,0.4,1.6,0.6,2.5,0.5h2.8l22.5,0.2l11.3,0.1H70.5z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M68,92.1c-0.7,3.4-2.7,6.4-5.6,8.4c-2.9,1.9-6.5,2.7-10,2C45.1,101,40.3,94,41.6,86.7c0.6-3.5,2.6-6.6,5.5-8.7 c2.9-2.1,6.5-3,10-2.4c3.5,0.5,6.7,2.4,8.8,5.2c1,1.4,1.8,3,2.3,4.7c0.4,1.7,0.5,3.4,0.3,5.1c0,0.1-0.1,0.2-0.2,0.2 c-0.1,0-0.2-0.1-0.2-0.2c0.3-3.3-0.7-6.7-2.8-9.3c-2-2.6-5-4.3-8.3-4.7c-3.2-0.4-6.5,0.4-9.2,2.3s-4.4,4.9-5,8.1 c-0.6,3.2,0.1,6.6,2,9.3c1.9,2.7,4.7,4.6,7.9,5.3c4.9,1,10-1,13-5C66.7,95.3,67.5,93.7,68,92.1C67.9,92,67.9,92,68,92.1L68,92.1z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M63.6,91.9c-1.4,4.8-6.4,7.6-11.2,6.2c-2.1-0.6-3.9-1.9-5.1-3.8c-2.8-4.1-1.9-9.6,2-12.6 c1.8-1.5,4.2-2.1,6.5-1.9c2.3,0.2,4.5,1.3,6.1,3c1.5,1.7,2.4,4,2.3,6.3c0,0.1-0.1,0.2-0.2,0.2s-0.2-0.1-0.2-0.2 c-0.1-2.1-1-4.1-2.4-5.7c-1.5-1.5-3.5-2.5-5.6-2.6c-2.1-0.2-4.2,0.5-5.8,1.8c-1.6,1.3-2.7,3.2-3,5.2s0.1,4.1,1.2,5.9 c1.2,1.8,2.9,3,5,3.5c2.1,0.5,4.2,0.3,6.1-0.7C61.2,95.5,62.7,93.8,63.6,91.9c-0.1-0.2-0.1-0.2,0-0.2V91.9z M32.5,106l-13,0.3 c-0.6,0-1.2-0.1-1.8-0.3c-0.5-0.3-1-0.8-1.3-1.3c-0.2-0.6-0.3-1.2-0.3-1.8v-1.6L16,94.8V78.5c-0.1-1.1,0-2.2,0.1-3.3 c0.3-2.3,1.5-4.4,3.3-5.9c1.8-1.5,4-2.3,6.3-2.4c1.1-0.1,2.3,0.1,3.4,0.4c1.1,0.3,2.2,0.9,3.1,1.6c1.8,1.5,3,3.6,3.3,5.9 c0.1,1.2,0.1,2.2,0.1,3.3v3.3l-0.1,13l-0.1,6.5v1.6c0,0.6,0,1.1-0.1,1.7C34.8,105.4,33.7,106.1,32.5,106z M32.5,105.7 c1,0,1.9-0.7,2.2-1.7c0.1-0.5,0-1,0.1-1.6v-1.6l-0.1-6.5l-0.1-13V78c0-1.1,0-2.2-0.1-3.2c-0.2-2-1.3-3.9-2.9-5.2 c-0.8-0.6-1.7-1.1-2.7-1.4c-1-0.3-2.1-0.4-3.1-0.3c-4.2,0-7.8,3.1-8.5,7.3c-0.1,1.1-0.2,2.1-0.1,3.2v16.3l-0.1,6.5v1.6 c0,0.5,0,1,0.2,1.4c0.2,0.4,0.5,0.8,0.9,1c0.4,0.2,0.9,0.3,1.4,0.3C23.8,105.6,28.2,105.7,32.5,105.7z M69.2,100.2 c0.5-0.2,1,0,1.3,0.4v0.1c0.3,0.5,0.2,1.2-0.3,1.6c-0.4,0.4-1.1,0.5-1.6,0.3c-0.5-0.2-0.8-0.8-0.6-1.3c0-0.1,0.1-0.2,0.2-0.1h0.1 c0.1,0,0.1,0,0.1,0.1c0.4,0.5,0.8,0.5,1,0.3c0.2-0.2,0.2-0.3,0.1-0.4c-0.1-0.1-0.1-0.4-0.4-0.7C69.1,100.3,69.1,100.3,69.2,100.2z  M44.3,77.4c-1.3,0.3-2.7,0.4-4,0.4h-1.6c-0.3,0-0.6-0.2-0.8-0.5c-0.1-0.2-0.2-0.5-0.2-0.8v-1.6c0-0.3,0.2-0.6,0.5-0.8 c0.2-0.1,0.5-0.2,0.8-0.2h4.5c0.2,0,0.5,0.1,0.7,0.2c0.2,0.2,0.4,0.4,0.4,0.7v0.6c0,0.3-0.1,0.7-0.1,1c0,0.1-0.1,0.2-0.2,0.2 c-0.1,0-0.2-0.1-0.2-0.2c0-0.3-0.1-0.7-0.1-1V75c0-0.1,0-0.1-0.1-0.1h-5v1.6h1.5c1.3,0,2.7,0.1,4,0.4C44.3,77.2,44.4,77.2,44.3,77.4 z" >
    </path>

    <path
        android:fillColor="@color/grey_lighten_10"
        android:pathData="M18.4,70.1c0.9-0.1,1.7,0,2.5,0.4c0.7,0.4,1.3,1,1.8,1.7c0.4,0.6,1,1,1.6,1.4c0.7,0.3,1.4,0.4,2.1,0.4 c0.7,0,1.4-0.1,2.1-0.4c0.7-0.3,1.3-0.8,1.8-1.3c0.6-0.6,1.3-1.1,2-1.6c0.8-0.4,1.6-0.6,2.5-0.5c0.1,0,0.2,0.1,0.2,0.2 s-0.1,0.2-0.2,0.2c-0.7,0.1-1.5,0.3-2.1,0.7c-0.6,0.5-1.2,1-1.7,1.6c-0.6,0.6-1.3,1.2-2,1.6c-1.6,0.8-3.5,0.8-5.1,0.1 c-0.8-0.4-1.4-1.1-1.9-1.8c-0.4-0.6-0.9-1.2-1.5-1.7C19.8,70.6,19.1,70.3,18.4,70.1C18.3,70.3,18.3,70.3,18.4,70.1 C18.3,70.2,18.4,70.2,18.4,70.1z M76.4,75.9c0.2,1.9-0.2,3.8-0.9,5.5c-0.7,1.6-1.1,3.4-1.1,5.2c-0.1,1.8,0.3,3.5,1,5.1 c0.4,0.9,0.7,1.8,0.9,2.7c0.2,0.9,0.3,1.9,0.2,2.8c0,0.1-0.1,0.2-0.2,0.2c-0.1,0-0.2-0.1-0.2-0.2c-0.1-0.9-0.3-1.8-0.6-2.6 c-0.3-0.9-0.7-1.7-1.1-2.5c-0.8-1.7-1.3-3.6-1.3-5.5c0-1,0.1-2,0.4-2.9c0.2-0.9,0.6-1.8,0.9-2.7c0.3-0.9,0.7-1.7,1-2.5 c0.3-0.8,0.5-1.7,0.6-2.6C76.2,75.8,76.3,75.8,76.4,75.9z M16.6,82c0.1,2.2-0.2,4.4-0.9,6.5s-1.9,4-3.5,5.6c-1.6,1.5-3.3,2.8-5.2,4 c-1.8,1-3.4,2.6-4.4,4.4c-0.8,1.9-0.9,4-0.1,5.9s2.3,3.4,4.2,4.2c1.9,0.9,4,1.4,6.1,1.7c2.1,0.3,4.3,0.4,6.4,0.4 c8.6-0.2,17.2-1.1,25.7-2.9c8.5-1.6,16.9-3.7,25.3-6c0.1,0,0.2,0,0.2,0.1s0,0.2-0.1,0.2c-4.1,1.3-8.3,2.5-12.5,3.6 c-4.2,1.1-8.4,2.1-12.7,3c-4.3,0.9-8.6,1.6-12.9,2.2s-8.7,0.9-13.1,0.9c-2.2,0-4.4-0.1-6.6-0.4c-2.2-0.2-4.4-0.9-6.4-1.8 c-2.1-0.9-3.8-2.7-4.7-4.8c-0.4-1.1-0.6-2.2-0.6-3.4c0-1.2,0.2-2.3,0.7-3.4c1-2.1,3-3.5,4.8-4.7c1.8-1.2,3.6-2.4,5.2-3.8 c1.5-1.4,2.7-3.2,3.5-5.2c0.7-2,1.1-4.1,1.2-6.3C16.4,81.9,16.5,81.9,16.6,82z" >
    </path>

</vector>
```

Reply:
Sure thing, thanks for the SVG! Done with 871d891.